getting-started/create-a-mobile-property.md

```diff
@@ -13,7 +13,8 @@ Having trouble creating a mobile property or need access to Experience Platform
 
    If necessary, you can change the [**Privacy** ](../resources/privacy-and-gdpr.md#setting-privacy-status) and **HTTPS** settings later.
 
-3. Locate the new property in the **Properties** list and open it.
+3. Click **Save** to create the mobile property.
+4. Locate the new property in the **Properties** list displayed and open it.
```

Comment:
Can you please reword step 4 as follows:
"4. Locate the new property in the displayed **Properties** list and open it."